### DISCUSSION TITLE
sh download_pretrained_models.sh errors

### DISCUSSION THREAD

**Discussion by LizChudleigh** [AI-sandbox/gnomix#49] [General]
Hi, It looks like the sh download_pretrained_models.sh code is no longer working due to the new google virus scan messaging Google Drive can't scan this file for viruses.

[pretrained_gnomix_models.tar.gz](https://drive.usercontent.google.com/open?id=1Q0zg9zqTaZUvt42uxzE_0gcfnFvjwi33&authuser=1) (2.3G) is too large for Google to scan for viruses. Would you still like to download this file?

Was able to go to the link directly, but just wanted to give a headsup in case anyone wants to update download_pretrained_models.sh

**Comment by oeco28** [comment]
Hello @LizChudleigh , I have modified their download_pretrained_models.sh

you can use this code instead:

```
#!/usr/bin/env bash
# Modern replacement for download_pretrained_models.sh
pip install -q gdown

# File ID from original download_pretrained_models.sh
FILEID="1Q0zg9zqTaZUvt42uxzE_0gcfnFvjwi33"

echo "Downloading pretrained gnomix models..."
gdown "${FILEID}" -O pretrained_gnomix_models.tar.gz

echo "Expanding files..."
tar -xvf pretrained_gnomix_models.tar.gz
rm pretrained_gnomix_models.tar.gz
echo "Done. Models in ./pretrained_gnomix_models"
```

This should do the trick.
